### PR TITLE
Mastery Exercise submission

### PR DESCRIPTION
**Pull request by daveharmswebdev**

### CODE REVIEW

**Comment by daveharmswebdev** [comment]
Can you check this please. I believe I am finished.

File: word-search.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    .pipe(transStream)
+    .pipe(writeStream)
+
+  // writestream code
+  writeStream._write = (buffer, _, cb) => {
+    let output = `${buffer.toString()}`
+    process.stdout.write(`${output}`)
+    cb()
+  }
+  // readStream.pipe(transStream).pipe(writeStream)
```
Comment (morecallan) [inline word-search.js]: You can drop this commented out code before merge.

**Comment by morecallan** [comment]
1. YES. Commits are broken down into easily parsed tasks. +1 coolPoints.
2. Meeting MVP requirements, showing solid grasp of the material. +1 coolPoints
3. Modular and concise. +1 coolPoints

To revise before merge:
1. Comments! 
2. Remove commented out code.

Some fun tips:
1. ES6 has a nice fun .startsWith method on the string prototype that might be handy (also a .includes)
2. stdout is a writable stream so you can pipe directly to it without having to create a new writable.

**Comment by daveharmswebdev** [comment]
@morecallan removed the commented code.

**Comment by morecallan** [comment]
@daveharmswebdev feel free to merge. 

Just making sure you get the comment about the stdout stream first?

**Pull request merged**